lib/pages/chat_list/chat_list.dart

```diff
@@ -788,7 +788,6 @@ class ChatListController extends State<ChatList>
     WidgetsBinding.instance.addPostFrameCallback((_) async {
       if (mounted) {
         Matrix.of(context).backgroundPush?.setupPush();
-        await matrixState.retrievePersistedActiveClient();
```

Comment:
why we remove it? any impaction?

Reply:
i have moved it to  `AppAdaptiveScaffoldBodyController` since its executed before this, and we need to have the correct client for the avatar to be in sync